packages/components/src/components/link/link.stories.ts

```diff
@@ -32,297 +39,119 @@ export const Default = {
-      }
-    })
-};
-
-/**
- * Use the `icon-left` and `icon-right` slots to add icons. They automatically adapt the size.
+ * Use the `standalone` attribute to control the layout of the icon and text within the component.
+ * If true, the icon and text will be displayed side by side, each occupying its own column.
+ * If false or not provided, the icon will be displayed inline within the text.
  */
 
-export const SizeAndIconSlots = {
-  name: 'Size × Icon Slots',
-  parameters: { controls: { exclude: ['default', 'icon-left', 'size', 'icon-right', 'standalone'] } },
-  render: (args: any) => {
-    return html`
-      ${[false, true].map(standalone =>
-        generateTemplate({
-          axis: {
-            x: {
-              // To make the story creation easier, we put everything in the default slot.
-              type: 'slot',
-              name: 'default',
-              title: 'slot="icon-..."',
-              values: [
-                { value: 'Link', title: '–' },
-                {
-                  value: 'Link<sd-icon library="global-resources" name="system/picture" slot="icon-left"></sd-icon>',
-                  title: 'left'
-                },
-                {
-                  value: 'Link<sd-icon library="global-resources" name="system/picture" slot="icon-right"></sd-icon>',
-                  title: 'right'
-                }
-              ]
-            },
-            y: { type: 'attribute', name: 'size' }
-          },
-          options: { title: `standalone="${standalone ? 'true' : 'false'}"` },
-          constants: { type: 'attribute', name: 'standalone', value: standalone },
-          args
-        })
-      )}
-    `;
-  }
+export const Standalone = {
+  name: 'Standalone',
+  render: () => html`
+    <div class="flex gap-12">
+      <div class="sd-prose">
+        <ul>
+          <li>
+            <sd-link href="http://union-investment.com" standalone>
+              <sd-icon library="global-resources" name="system/home" slot="icon-left"></sd-icon>
+              Union Investment
+            </sd-link>
+          </li>
+          <li>
+            <sd-link href="https://solid-design-system.fe.union-investment.de/x.x.x/storybook/" standalone>
+              <sd-icon library="global-resources" name="system/pen" slot="icon-left"></sd-icon>
+              Solid Design System
+            </sd-link>
+          </li>
+        </ul>
+
+        <p>
+          While the list above shows <code>standalone</code> links, we now will link to the
+          <sd-link href="https://cd.union-investment.de">CD Toolbox</sd-link> inside a paragraph.
+        </p>
+      </div>
+    </div>
+  `
 };
 
 /**
- * Icons automatically adapt to the link's invertedness.
+ * Use the `size` attribute to adjust the size of the link. By default it is set to `inherit` to adapt to the surrounding text but can be explicitly changed to `lg` or `sm`.
```

Comment:
* Use the `size` attribute to adjust the size of the link. By default it is set to `inherit` to adapt to the surrounding text but can be changed to `lg` or `sm`.

Reply:
Updated